### ISSUE TITLE
scrcpy: error while loading shared libraries: libavformat.so.57: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by hunterx9** [Genymobile/scrcpy#1025]
Was working find in ubuntu 18.04 but when i updated to ubuntu 19.04 it shows the error. Any solution for this?

**Comment by rom1v** [comment]
(you need to rebuild from scratch)

**Comment by pconrad-fb** [comment]
If there's a command `apt install scrcpy` that installs `scrcpy` in my distro (there is) and a distro upgrade breaks `scrcpy` (it does) then "You need to rebuild from scratch" is just going to prevent a lot of people from using `scrcpy`.

**Comment by rom1v** [comment]
> and a distro upgrade breaks `scrcpy`

A distro upgrade probably does not break. The reason is probably becaused you installed a version manually in your old distro.

If you ever only installed it via `apt`, you should never face this issue.

**Comment by pconrad-fb** [comment]
If that's the case, how can I remove that version? That's good news if true.

**Comment by rom1v** [comment]
If you installed with the [simple](https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#simple) installation script, you can `sudo ninja -Cbuild-auto uninstall`.

Otherwise, you can remove manually the file installed: https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#install

**Comment by pconrad-fb** [comment]
Well, shoot, that's excellent. I apologize for coming in all annoyed!

I did find it really hard to uninstall. `ninja` kept telling me it couldn't find the directory (I had `scrcpy` in `usr/local/bin`) so I ended up flailing a bit. I removed the file manually then did `sudo apt purge scrcpy` then did `hash -r` to get Bash to forget, then I was able to install with `apt`. Thanks! I'm back on! It's a very useful program, and I would hate to have it unavailable.
